app/src/main/java/com/locoquest/app/MainActivity.kt

```diff
@@ -94,6 +94,7 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
      * Called when the activity receives a result from another activity.
      * In this case, it handles the result of the Google One-Tap sign-in dialog.
      */
+    @Deprecated("Deprecated in Java")
```

Comment:
Added deprecated tag as recommended, this method is no longer in use

Reply:
Is there a better option to use now?  A recommended alternative?

Reply:
From what I found, ActivityResultLauncher. I tried to implement it myself, but ran into some issues. Here's a resource I was using: https://stackoverflow.com/questions/62671106/onactivityresult-method-is-deprecated-what-is-the-alternative